django_fido/js/fido2.js

```diff
@@ -77,8 +77,15 @@ function fido2ErrorResponseCallback(error) {
     window.dispatchEvent(new Event('on-close-fido-window'))
 }
 
-async function sendFido2Request(url, is_registration, credenetials_name) {
-    const response = await fetch(url)
+async function sendFido2Request(url, is_registration, credenetials_name, csrf_token, form_data = '') {
+    const response = await fetch(url, {
+        method: 'POST',
```

Comment:
I don't think it's such a dangerous operation, that it would need POST.

Reply:
We agreed on changing the API back to GET. Updated.